src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/models/ClaimantDetailsContent.java

```diff
@@ -12,7 +10,7 @@
 
 import static uk.gov.hmcts.cmc.claimstore.utils.Formatting.formatDate;
 
-public class DefendantDetailsContent {
+public class ClaimantDetailsContent {
```

Comment:
That class is exactly the same as `uk.gov.hmcts.cmc.claimstore.documents.content.models.PartyDetailsContent#PartyDetailsContent` - please remove.